spec/Section 4 -- Introspection.md

```diff
@@ -421,5 +430,7 @@ Fields
   locations this directive may be placed.
 * `args` returns a List of `__InputValue` representing the arguments this
   directive accepts.
+  * Accepts the argument `includeDeprecated` which defaults to {false}. If
```

Comment:
The SDL definition of the `__Directive.args` field on line 192 of this page needs to be updated to reflect the new `includeDeprecated` argument.